docs/SupportedONNXOps-cpu.md

```diff
@@ -49,7 +49,7 @@ Onnx-mlir currently supports ONNX operations targeting up to opset 20. Limitatio
 | **Concat** |6 - * | | |
 | **ConcatFromSequence** |none | | | |
 | **Constant** |6 - * | | |
-| **ConstantOfShape** |9 - 19 | | |
+| **ConstantOfShape** |20 - * | | |
```

Comment:
The first number is the lowest opset we support for an operator. So that should stay as `9` here unless something happened that we no longer support this operator in older Opsets. 

If that was the case (which I don't think it is), then `# ==MIN== 9` would need to be updated for this Op in inference_backend.py as well.

Reply:
> The first number is the lowest opset we support for an operator. So that should stay as `9` here unless something happened that we no longer support this operator in older Opsets.
> 
> If that was the case (which I don't think it is), then `# ==MIN== 9` would need to be updated for this Op in inference_backend.py as well.

Thanks, I will open a new PR and update `inference_backend.py` and `SupportedONNXOps-cpu.md`